source/ja/quickstart.rst

```diff
@@ -46,13 +46,13 @@ RHEL 6 では、依存パッケージ (``oniguruma``) のインストールに `
 ::
 
   // RHEL 6 の場合
-  $ sudo yum install jubatus-plugin-mecab jubatus-plugin-ux
+  $ sudo yum install jubatus-plugin-mecab jubatus-plugin-ux jubatus-plugin-image
 
   // RHEL 7 の場合
   $ sudo yum install jubatus-plugin-mecab jubatus-plugin-ux jubatus-plugin-image
```

Comment:
I think we don't have to separate steps for "RHEL 6" and "RHEL 7".

```
$ sudo yum install jubatus-plugin-mecab jubatus-plugin-ux jubatus-plugin-image
```

works fine for both.